pkg/debian9/Dockerfile

```diff
@@ -152,12 +152,15 @@ ENV LD_LIBRARY_PATH=/opt/hubble/lib:/lib:/lib64:/usr/lib:/usr/lib64:/usr/local/l
 RUN git clone "$HUBBLE_GIT_URL" "$HUBBLE_SRC_PATH" \
  && cd "$HUBBLE_SRC_PATH" \
  && git checkout "$HUBBLE_CHECKOUT" \
- && cp -rf "$HUBBLE_SRC_PATH" /hubble_build
+ && cp -rf "$HUBBLE_SRC_PATH" /hubble_build \
```

Comment:
Values of 'branch' and 'last_commit' are being updated here.
Since we don't need the commit when we are using a tag, I have used 'TAGGED_BUILD' string here.